app/javascript/components/ChatWindow.jsx

```diff
@@ -33,20 +33,45 @@ const Message = ({
   sent,
   timestamp
 }) => (
-  <li className={sent ? "message-sent" : "message-received"}>
+  // <li className={sent ? "message-sent" : "message-received"}>
     <Paper
       zDepth={2}
       className={"message-bubble " + (sent ? "message-sent" : "message-received")} >
-      <div className="content-text">
-        {text}
-      </div>
-      <div className="content-timestamp">
-        {moment(timestamp).format("HH:mm")}
-      </div>
+      <li>
+        <div className="content-text">
+          {text}
+        </div>
+        <div className="content-timestamp">
+          {moment(timestamp).format("HH:mm")}
+        </div>
+      </li>
     </Paper>
-  </li>
+  // </li>
 )
 
+const MessageBulk = ({
+  messages
+}) => {
+  console.log(messages)
+  const sentMessages = messages[0].sent
+  return (
+    <Paper
+      zDepth={2}
+      className={"message-bubble " + (sentMessages ? "message-sent" : "message-received")} >
+      {messages.map(message =>
+        (<li key={message.id}>
```

Comment:
this seems to no longer use <Message>, the component could be removed (or if used, remove the old code commented in it)